sdcard/RES/FREQMAN/THIS IS DEMO FOLDER

```diff
@@ -0,0 +1,4 @@
+Please put your personal frequency lists into /USR/FREQMAN
+The system will automatically handle and list both dir together
+So you can easily updating official resources
+without manually handling your personal files
```

Comment:
I'd suggest:

```
Please put your personal frequency lists
in the /USR/FREQMAN folder to avoid them
being overwritten when updating SD Card
contents. Firmware apps will check in both
locations for frequency list files.
```

Reply:
Thank you very much! 
Suggestion accepted.